.github/workflows/release.yml

```diff
@@ -0,0 +1,23 @@
+name: "release"
+on:
+  push:
+    tags:
+      - 'v*'
```

Comment:
What's the convention for versions here? SEMVER e.g. `v1.2.3`?

It might be a good time for us to decide on an nvim-tree version scheme @gegoune

Reply:
I'm not sure if it's a [hard requirement](https://github.com/luarocks/luarocks/wiki/Rockspec-format), but the convention for luarocks is semver.

Reply:
Thank you.

Reply:
vMajor.Minor.Patch seems to be standard. Lets use that.